CMakeLists.txt

```diff
@@ -17,6 +17,8 @@
 #
 
 cmake_minimum_required(VERSION 2.6)
+cmake_policy(SET CMP0011 NEW)
```

Comment:
Checking my understanding: this is necessary to avoid a warning when interpreting the following line?

Reply:
This is necessary to avoid a warning when using any FindPython* on cmake > 3.18. The warning indicates this policy has to be set one way or the other, having it set to default is deprecated.